mgr/Conscript-standard

```diff
@@ -197,7 +197,8 @@ if ( $pkg eq "Jevp") {
 	      .   $main::PATH_SEPARATOR . "#OnlTools/Jevp/StJevpBuilders"
 	      .   $main::PATH_SEPARATOR . "#OnlTools/Jevp/StJevpViewer"
 	      .   $main::PATH_SEPARATOR .  $CPPPATH;
-    $CPPFLAGS .= " -DQT_THREAD_SUPPORT -DQT_SHARED -DQT_NO_DEBUG ";
+    $CPPFLAGS .= " -DQT_THREAD_SUPPORT -DQT_SHARED -DQT_NO_DEBUG -DRTS_ENABLE_LOG ";
```

Comment:
Jevp gets compiled when anyone checks out and compiled this `star-sw` repository (including official library compilations at SDCC currently, e.g. $STAR/.sl73_gcc485/LIB/libJevp.so). Does this mean the logging will be enabled within Jevp for everyone? Is that the desired effect?

Reply:
regarding SL23b/SL23c I don't mind if it has no effect.  it works for SL23c and it was the newest so its what I chose.

regarding the desired effect.  Yes, that is the desired effect.   The server is a rather special case, I wouldn't expect anyone to attempt to run it other than on the official servers.    For the builders, which users do run while debugging, the default is set to write to stderror, so in order to see any output the users need to have logging enabled.

Reply:
Is there a way to set a define easily at the compile step?   via a cons flag or environment variable?   If so, I would be happy to use such a mechanism, rather than set it globally.